torchtune/models/llama3_1/__init__.py

```diff
@@ -7,10 +7,13 @@
 from ._component_builders import llama3_1, lora_llama3_1
```

Comment:
linter unhappy : ( need to add to `__all__`?

Reply:
It's in __all__, no idea what's triggering this. The pre-commit flake8 passes too